src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String[]> readFromFile(String pathToFile) {
```

Comment:
```suggestion
    public List<String> readFromFile(String pathToFile) {
```
dont split the data, just return the line
then split in parser